http/dashboard_service.go

```diff
@@ -210,6 +210,32 @@ type dashboardCellResponse struct {
 	Links map[string]string `json:"links"`
 }
 
+func (d *dashboardCellResponse) MarshalJSON() ([]byte, error) {
```

Comment:
can you assign the properties directly to the dashboard cell response instead? extending the `dashboardCellResponse` directly on line 209? Think you're better off making this type more xplroable than hiding deets in what is likely unneccssary the MarshalJSON method 🤷‍♂

Reply:
I hear you - the main reason for me doing this was because I didn't know how to get the `dashboardCellResponse` to surface up the links after adding the view properties to each cell.. Ideally we shouldn't have to marshal the `dashboardCellResponse`, but in my tests I keep getting:
```go
               "cells": [
                 {
                   "h": 4,
                   "id": "da7aba5e5d81e550",
                   "name": "the cell name",
                   "properties": {
                     "axes": null,
                     "colors": null,
                     "geom": "",
                     "legend": {
                     },
                     "note": "",
                     "queries": null,
                     "shadeBelow": false,
                     "showNoteWhenEmpty": false,
                     "type": "xy",
                     "xColumn": "",
                     "yColumn": ""
                   },
                   "w": 3,
                   "x": 1,
                   "y": 2
            +      "links": {
            +        "self": "/api/v2/dashboards/020f755c3c082000/cells/da7aba5e5d81e550",
            +        "view": "/api/v2/dashboards/020f755c3c082000/cells/da7aba5e5d81e550/view"
            +      }
                 }
               ],
```

the code I added in the MarshalJSON adds the links, although it's the same as doing... 
```go
type dashboardCellResponse struct {
	platform.Cell
	Properties platform.ViewProperties `json:"properties,omitempty"`
	Links      map[string]string       `json:"links"`
}
```

Reply:
@dearyhud lets chat after standup, I'm not sure I follow the issue here 🤔

Reply:
sure